### PR TITLE
test: adding maps color unit test

### LINKED ISSUE #1982: Test/Followup: Write test for GeoOptions using properties
See: #1951

We fixed a bug with GeoOptions, but never wrote a test for it. This is the issue to follow up and write the test so that we don't have any regressions in that regard.

### Acceptance Criteria
- For the `Maps/view.tsx` tests, there are two scenarios tested:
  - That by default, proper colors are passed into`layers`
  - That the default can be overriden

### PR DESCRIPTION
**Pull request by zoesteinkamp**
Closes #1982

### CODE REVIEW

File: cypress/e2e/shared/flows.test.ts [hunk 1]
```diff
@@ -370,4 +370,88 @@ describe('Flows', () => {
...
+    cy.getByTestID('tag-selector cool').click()
+    cy.getByTestID('time-machine-submit-button').click()
+
+    // we should only see cool in the table
+    cy.getByTestID('simple-table').should('be.visible')
+    cy.getByTestID('table-cell cool')
+      .first()
+      .should('be.visible')
+    cy.getByTestID('table-cell beans').should('not.exist')
+  })
```
Comment (hoorayimhelping) [inline cypress/e2e/shared/flows.test.ts:456]: ✂️

Comment (zoesteinkamp) [reply cypress/e2e/shared/flows.test.ts]: got rid of these

**Review by Palakp41 — APPROVED** [review]
LGTM!

**Comment by ui-ci-bot-influx** [comment]
Metric | Score | Level
------------ | ------------- | ------------- 
performance | 61 | :yellow_circle:
accessibility | 96 | :green_circle:
seo | 90 | :green_circle:
pwa | null | :hankey:
best-practices | 100 | :green_circle:


[Compare your scores with master branch](https://influxdb.aws.influxdata.io/orgs/844910ece80be8bc/dashboards/073c4b95f1089000?lower=now%28%29%20-%202d&vars%5Blighthouse_score_branch%5D=2106)

**Review by hoorayimhelping — APPROVED** [review]

**Pull request merged**